commerce-test/src/testIntegration/java/com/liferay/commerce/internal/order/test/CommerceOrderHttpHelperImplTest.java

```diff
@@ -111,11 +122,23 @@ public void setUp() throws Exception {
 		_httpServletRequest.setAttribute(WebKeys.THEME_DISPLAY, _themeDisplay);
 
 		_themeDisplay.setRequest(_httpServletRequest);
+
+		_commerceOrders = new ArrayList<>();
 	}
 
 	@After
-	public void tearDown() {
+	public void tearDown() throws PortalException {
 		CentralizedThreadLocal.clearShortLivedThreadLocals();
+
+		for (CommerceOrder commerceOrder : _commerceOrders) {
+			CommerceOrderLocalServiceUtil.deleteCommerceOrder(commerceOrder);
+		}
+
+		CommerceDiscountLocalServiceUtil.deleteCommerceDiscounts(
+			_group.getCompanyId());
+		CommerceAccountLocalServiceUtil.deleteCommerceAccount(_commerceAccount);
+		GroupLocalServiceUtil.deleteGroup(_group);
+		UserLocalServiceUtil.deleteUser(_user);
```

Comment:
Is there a reason we're using `*LocalServiceUtil` here and not injecting the `*LocalService` instances instead? Seems like we should keep consistent by using the instances as much as possible.